ruby/private/toolchains/repository_context.bzl

```diff
@@ -4,7 +4,7 @@
 #
 
 def _eval_ruby(ruby, script, options = None):
-    arguments = ["env", "-i", ruby.interpreter_realpath]
+    arguments = [ruby.interpreter_realpath]
```

Comment:
Did you really need to remove `env -i` from this place?

Reply:
I think this was an experiment to see if building gems with native extensions on Linux was possible. Our theory was that env -i was responsible for crashes during compilation of the native gems on CI.

I didn’t realize this was merged so quickly, but I have a different theory as to why the gems aren’t building natively on CI, but they do build locally on OSX. If my theory proves correct we may want to revert this PR.

My current theory is that the native extensions compilation together with the aggressive java flags resulted in out of memory error. I had to fight it yesterday when implementing circleci. Let me find a reference....

Reply:
This is the error we've been seeing on Linux / CI only:

https://stackoverflow.com/questions/11912878/gcc-error-gcc-error-trying-to-exec-cc1-execvp-no-such-file-or-directory

There are any answers on that page but none that mention out of memory error. And then I found a link from Alex Eagle about this issue and saying that have argrs should be set lower and number of jobs lower. Let me confirm that this indeed resolves the issue without removing env -i